fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/ParameterDAO.java

```diff
@@ -1,24 +1,27 @@
 /*
- * (C) Copyright IBM Corp. 2017,2019
+ * (C) Copyright IBM Corp. 2017,2018,2019
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 package com.ibm.fhir.persistence.jdbc.dao.api;
 
+import java.sql.Array;
+import java.sql.Connection;
 import java.util.List;
+import java.util.Map;
 
+import com.ibm.fhir.persistence.exception.FHIRPersistenceException;
 import com.ibm.fhir.persistence.jdbc.dto.Parameter;
 import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDBConnectException;
 import com.ibm.fhir.persistence.jdbc.exception.FHIRPersistenceDataAccessException;
 
 /**
- * This Data Access Object interface defines methods for creating, updating, and retrieving rows in the FHIR Parameter table.
- * @author markd
```

Comment:
should we remove the author?

Reply:
IMHO we shouldn't be using @author tags everywhere.  I've touched a lot of classes and I don't add my name everywhere.  I tried to remove it from places that have changed since the original authoring.  But we can discuss as a group.

Reply:
we just discussed this as a team and agreed that the `@author` tags add limited value beyond what git history already provides. furthermore, they get stale over time and become maintenance burden.  we will remove them incrmentally over time (as we edit classes) and this PR is consistent with that approach